src/main/java/net/neoforged/neoforge/common/Tags.java

```diff
@@ -572,6 +585,18 @@ public static class Items {
         public static final TagKey<Item> ORES_IN_GROUND_STONE = tag("ores_in_ground/stone");
         public static final TagKey<Item> PLAYER_WORKSTATIONS_CRAFTING_TABLES = tag("player_workstations/crafting_tables");
         public static final TagKey<Item> PLAYER_WORKSTATIONS_FURNACES = tag("player_workstations/furnaces");
+        /**
+         * Items that can hold various potion effects by making use of {@link net.minecraft.core.component.DataComponents#POTION_CONTENTS}.
+         * Contents of this tag may not always be a kind of bottle. Buckets of potions could go here.
+         * The subtags would be the name of the container that is holding the potion effects such as `c:potions/bucket` or `c:potions/vial` as examples.
+         */
+        public static final TagKey<Item> POTIONS = tag("potions");
+        /**
+         * Variations of the potion bottle that can hold various effects by using {@link net.minecraft.core.component.DataComponents#POTION_CONTENTS}.
+         * Examples are splash and lingering potions from vanilla.
+         * If a mod adds a new variant like a seeking potion that applies effect to the closest entity at impact, that would in this tag.
+         */
+        public static final TagKey<Item> POTION_BOTTLE = tag("potions/bottle");
```

Comment:
Seems like other tags retain the "parent" tag name in their names, ie `CLUMPS_RESIN` or `BRICKS_RESIN`. I think keeping that pattern here makes sense.
```suggestion
        public static final TagKey<Item> POTIONS_BOTTLE = tag("potions/bottle");
```